index.js

```diff
@@ -22,6 +22,13 @@ var pkgUp = require('pkg-up')
 var testExclude = require('test-exclude')
 var yargs = require('yargs')
 
+var ProcessInfo
+try {
+  ProcessInfo = require('./lib/process.covered.js')
```

Comment:
thanks for doing this, would like us to get in the habit of splitting this repo into more modules.